packages/eslint-plugin-nimble/index.js

```diff
@@ -0,0 +1,203 @@
+const restrictedImportsPaths = () => [
+    {
```

Comment:
Can we align more with the patterns shown in the https://github.com/ni/javascript-styleguide
i.e. instead of creating a plugin, create a shareable config (we aren't trying to make new rules, just configure them): https://eslint.org/docs/latest/extend/shareable-configs

Package name: `@ni-private/eslint-config-nimble`

```
/README.js
/package.json
/javascript.js
/typescript.js
```